service/Abstractions/Pipeline/DataPipeline.cs

```diff
@@ -149,10 +149,7 @@ public void MarkProcessedBy(IPipelineStepHandler handler)
         /// <param name="text">Text to store for the end user</param>
         public void Log(IPipelineStepHandler handler, string text)
         {
-            if (this.LogEntries == null)
-            {
-                this.LogEntries = new List<PipelineLogEntry>();
-            }
+            this.LogEntries ??= new List<PipelineLogEntry>();
```

Comment:
```suggestion
            if (this.LogEntries == null)
            {
                this.LogEntries = new List<PipelineLogEntry>();
            }
```

reverting changes to reduce size of Abstractions changes